core/src/test/resources/com/adobe/aem/commons/assetshare/components/structure/impl/HeaderImplTest.json

```diff
@@ -0,0 +1,184 @@
+{
+	"jcr:primaryType" : "cq:Page",
```

Comment:
This should be the minimal set of content required to satisfy this test case; it appears like there’s a lot more content in here.

Reply:
We probably should form some sort of convention for orangizng JSON resource files ... ex. 1 file per test, or 1 file per test car and the json has multiple “test content trees” in it .. also some naming conventions.

Reply:
> This should be the minimal set of content required to satisfy this test case; it appears like there’s a lot more content in here.

:-) I guess I should add some more tests to justify this large JSON

Reply:
> We probably should form some sort of convention for orangizng JSON resource files ... ex. 1 file per test, or 1 file per test car and the json has multiple “test content trees” in it .. also some naming conventions.

I would rather call it guidelines. In the first place 1 JSON file per test car should be enough, but that depends on some other factors. For example you have content at a fixed location and you need to test different versions of that content in test cases.